src/encoded/tests/features/software.feature

```diff
@@ -0,0 +1,50 @@
+@title
+Feature: Title
+
+    Scenario: Title updates
+        When I press "Materials & Methods"
+        And I wait for the content to load
+        And I click the link to  "/encode-software"
+        And I wait for the content to load
+        Then the title should contain the text "ENCODE Software"
+
+@Content
+Feature: ENCODE Software
+    Background:
+        When I visit "/encode-software/?type=Software&used_by=DCC"
+        And I wait for the content to load
+
+    Scenario: ENCODE Software- Portal
+        Then the title should contain the text "ENCODE Software"
+        Then I should see "ENCODE Software"
+        Then I should see "Software relevant to Data Coordination Center (DCC)"
```

Comment:
I'm not sure if these tests are up-to-date with the demo. For instance, I see "Software implemented and developed by the DCC (Data Coordination Center) for the Portal" on the demo for this test, and I see "All software used or developed by the ENCODE Consortium" under the All button instead of "Software relevant to ENCODE in generalp" - which has a typo, so if it did say that, the typo would need to get fixed. Somehow all tests are passing, though, which is really confusing to me.